datafusion/sqllogictest/test_files/arrow_typeof.slt

```diff
@@ -384,4 +384,35 @@ LargeList(Field { name: "item", data_type: Int64, nullable: true, dict_id: 0, di
 query T
 select arrow_typeof(arrow_cast(make_array([1, 2, 3]), 'LargeList(LargeList(Int64))'));
 ----
-LargeList(Field { name: "item", data_type: LargeList(Field { name: "item", data_type: Int64, nullable: true, dict_id: 0, dict_is_ordered: false, metadata: {} }), nullable: true, dict_id: 0, dict_is_ordered: false, metadata: {} })
\ No newline at end of file
+LargeList(Field { name: "item", data_type: LargeList(Field { name: "item", data_type: Int64, nullable: true, dict_id: 0, dict_is_ordered: false, metadata: {} }), nullable: true, dict_id: 0, dict_is_ordered: false, metadata: {} })
+
+## FixedSizeList
+
+query ?
+select arrow_cast(null, 'FixedSizeList(1, Int64)');
+----
+NULL
+
+#TODO: arrow-rs doesn't support it yet
```

Comment:
Shouldn't we be casting `[1]` (not `1`)?

Reply:
I noticed that the List supports casting from UTF8 to List with a single size. Therefore, I think FixedSizeList should also support it. 
```
select arrow_cast('1', 'LargeList(Int64)');
----
[1]
```